examples/ColumnGroupsExample.js

```diff
@@ -65,6 +65,13 @@ class ColumnGroupsExample extends React.Component {
             width={150}
           />
         </ColumnGroup>
+        <Column
```

Comment:
Is this double header the desired behavior?  It seems weird to have the header twice.
<img width="288" alt="doubleheader" src="https://user-images.githubusercontent.com/1034455/28194643-aa8ca000-6813-11e7-980c-84454a8abb06.png">